aerospike/check.py

```diff
@@ -3,75 +3,173 @@
 
 # stdlib
 import socket
+import time
+import re
 from contextlib import closing
+from collections import namedtuple
 
 # project
 from checks import AgentCheck
 
-EVENT_TYPE = SOURCE_TYPE_NAME = 'aerospike'
-SERVICE_CHECK_NAME = '%s.server_up' % EVENT_TYPE
+SOURCE_TYPE_NAME = 'aerospike'
+SERVICE_CHECK_NAME = '%s.cluster_up' % SOURCE_TYPE_NAME
+CLUSTER_EVENT_TYPE = SOURCE_TYPE_NAME
+NAMESPACE_EVENT_TYPE = '%s.namespace' % SOURCE_TYPE_NAME
+NAMESPACE_TPS_EVENT_TYPE = '%s.namespace.tps' % SOURCE_TYPE_NAME
+SINDEX_EVENT_TYPE = '%s.sindex' % SOURCE_TYPE_NAME
 
+Addr = namedtuple('Addr', ['host', 'port'])
+
+def parse_sindex_namespace(data, namespace):
+    idxs = []
+    while data != []:
+        l = data.pop(0)
+
+        match = re.match('^ns=%s:[^:]+:indexname=([^:]+):.*$' % namespace,l)
```

Comment:
Just for reference, could you post in a small comment a small sample match line for a matching namespace index line?